src/xdocs/config_system_properties.xml

```diff
@@ -31,7 +31,108 @@
         defines ability use custom DTD files in config and load them from some location.
         The property type
         is <a href="property_types.html#boolean">boolean</a> and defaults
-        to <code>false</code>.
+        to <code>false</code>. Disabled by default due to security concerns.
+      </p>
+      <p>
+        Example is example of include xml files content in xml by ENTITY feature
```

Comment:
Why not add example section like checks? This way they can be linked directly.
Ex: https://checkstyle.org/config_annotation.html#AnnotationLocation_Examples

Also `Example is example of...` => `The following is an example of...`.
You should avoid a repetitive/circular statement like "A house is a house because it is a house.".

Reply:
done